static/app/views/performance/traces/tracesSearchBar.tsx

```diff
@@ -27,6 +42,8 @@ export function TracesSearchBar({
   const organization = useOrganization();
   const canAddMoreQueries = queries.length <= 2;
   const localQueries = queries.length ? queries : [''];
+  const supportedTags = getTracesSupportedTags();
```

Comment:
Did you want to memo this?

Reply:
nah it's a small dict comp, it shouldn't be a big deal